articles/openstack/ostack-how-static-ip.md

```diff
@@ -0,0 +1,175 @@
+| mtu                       | 1446                                 |
+| name                      | testnet                              |
+| port_security_enabled     | True                                 |
+| project_id                | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx     |
+| provider:network_type     | None                                 |
+| provider:physical_network | None                                 |
+| provider:segmentation_id  | None                                 |
+| qos_policy_id             | None                                 |
+| revision_number           | 3                                    |
+| router:external           | Internal                             |
+| segments                  | None                                 |
+| shared                    | False                                |
+| status                    | ACTIVE                               |
+| subnets                   |                                      |
+| tags                      |                                      |
+| updated_at                | 2018-09-05T12:20:30Z                 |
++---------------------------+--------------------------------------+
+```
+### Create a subnet and attach to the network
+```
+$ openstack subnet create --network testnet --subnet-range 10.1.1.0/24 testnet-sub
+
++-------------------+--------------------------------------+
+| Field             | Value                                |
++-------------------+--------------------------------------+
+| allocation_pools  | 10.1.1.2-10.1.1.254                  |
+| cidr              | 10.1.1.0/24                          |
+| created_at        | 2018-09-05T12:22:46Z                 |
+| description       |                                      |
+| dns_nameservers   |                                      |
+| enable_dhcp       | True                                 |
+| gateway_ip        | 10.1.1.1                             |
+| host_routes       |                                      |
+| id                | fc333721-47c8-4445-871b-d261eb195b41 |
+| ip_version        | 4                                    |
+| ipv6_address_mode | None                                 |
+| ipv6_ra_mode      | None                                 |
+| name              | testnet-sub                          |
+| network_id        | dd4f624a-1101-4b01-a388-308c5972742a |
+| project_id        | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx     |
+| revision_number   | 2                                    |
+| segment_id        | None                                 |
+| service_types     |                                      |
+| subnetpool_id     | None                                 |
+| tags              |                                      |
+| updated_at        | 2018-09-05T12:22:46Z                 |
++-------------------+--------------------------------------+
+```
+### Create a port and attach to the network & subnet ###
```

Comment:
3. Create a port and attach it to the network and subnet:

$ openstack port create --network testnet --fixed-ip subnet=testnet-sub,ip-address=10.1.1.10 test-static-ip-1

This returns the following: